sdk/react/controlBar.js

```diff
@@ -140,8 +141,10 @@ class ControlBar extends Component {
       android: this.props.config.controlBar.logo.imageResource.androidResource
     });
 
+    let castEnabled = this.props.config.castControls.enabled;
+    let color = this.props.inCastMode === true ? this.props.config.castDevicesScreen.iconStyle.active.color : this.props.config.castDevicesScreen.iconStyle.inactive.color;
```

Comment:
no need to compare it with `true`: `this.props.inCastMode ? ... : ...`

Reply:
Fixed. Thanks a lot!